src/mlpack/methods/ann/layer/noisylinear.hpp

```diff
@@ -48,6 +48,15 @@ class NoisyLinear
   //! Copy constructor.
   NoisyLinear(const NoisyLinear&);
 
+  //! Move constructor.
+  NoisyLinear(NoisyLinear&&);
+
+  //! Operator= copy constructor.
+  NoisyLinear& operator=(NoisyLinear const& layer);
```

Comment:
```suggestion
  NoisyLinear& operator=(const NoisyLinear& layer);
```
This is a super trivial comment but this is just to match the style in the rest of the codebase. :)